portal/src/main/webapp/css/patient-view/clinical-attributes.css

```diff
@@ -0,0 +1,131 @@
+/* Do not display clinical attributes on default */
+.clinical-attribute {
+    display: none;
+    order: 999;
+}
+/* Show only following attributes */
+.clinical-attribute[attr-id~="SEX"],
+.clinical-attribute[attr-id~="GENDER"],
+.clinical-attribute[attr-id~="AGE"],
+.clinical-attribute[attr-id~="OS_STATUS"],
+.clinical-attribute[attr-id~="OS_MONTHS"],
+.clinical-attribute[attr-id~="DFS_STATUS"],
+.clinical-attribute[attr-id~="DFS_MONTHS"],
+.clinical-attribute[attr-id~="CANCER_TYPE"],
+.clinical-attribute[attr-id~="CANCER_TYPE_DETAILED"],
+.clinical-attribute[attr-id~="KNOWN_MOLECULAR_CLASSIFIER"],
+.clinical-attribute[attr-id~="GLEASON_SCORE"],
+.clinical-attribute[attr-id~="HISTOLOGY"],
+.clinical-attribute[attr-id~="TUMOR_STAGE_2009"],
+.clinical-attribute[attr-id~="TUMOR_GRADE"],
+.clinical-attribute[attr-id~="ETS/RAF/SPINK1_STATUS"],
+.clinical-attribute[attr-id~="TMPRSS2-ERG_FUSION_STATUS"],
+.clinical-attribute[attr-id~="ERG-FUSION_ACGH"],
+.clinical-attribute[attr-id~="SERUM_PSA"],
+.clinical-attribute[attr-id~="DRIVER_MUTATIONS"],
+.clinical-attribute[attr-id~="SAMPLE_CLASS"],
+.clinical-attribute[attr-id~="SAMPLE_TYPE"],
+.clinical-attribute[attr-id~="DERIVED_SAMPLE_LOCATION"],
+#more-patient-info a .clinical_attribute[attr-id~="PRIMARY_SITE"] {
+    display: inline;
+    order: 10;
+}
+/* Hide following attributes for samples */
+/* Show comma before clinical attributes, except first one */
+.clinical-attribute:not(.first-order):before {
+    content: ", ";
+    color: #428bca;
+}
+/* Order clinical attributes */
+.clinical-attribute[attr-id~="PATIENT_DISPLAY_NAME"],
+.clinical-attribute[attr-id~="SAMPLE_DISPLAY_NAME"] {
+    order: 0;
+}
+.clinical-attribute[attr-id~="SEX"],
+.clinical-attribute[attr-id~="GENDER"] {
+    order: 1;
+}
+.clinical-attribute[attr-id~="AGE"] {
+    order: 2;
+}
+.clinical-attribute[attr-id~="OS_STATUS"] {
+    order: 3;
+}
+.clinical-attribute[attr-id~="OS_MONTHS"] {
+    order: 4;
+}
+.clinical-attribute[attr-id~="DFS_STATUS"] {
+    order: 5;
+}
+.clinical-attribute[attr-id~="DFS_MONTHS"] {
+    order: 6;
+}
+#more-patient-info a .clinical_attribute[attr-id~="PRIMARY_SITE"] {
```

Comment:
Or actually it only copies `CANCER_TYPE` and `CANCER_TYPE_DETAILED`, so maybe we should remove `PRIMARY_SITE` from the `more-patient-info` all together.